schultz_a07b.js

```diff
@@ -0,0 +1,93 @@
+/*Chris Schultz
+schultz_a07b.js
+Info 2124 WW
+Thoendel
+02-02-2020*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "cheese", "shrimp", "tomatoes");   //Modified line as per Step 1   
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    clearScreen();  //clearScreen is invoked regardless of next steps; let's do it first
+    if (text.toUpperCase() == "Q")  {//if input == Q || q....

```

Comment:
good use of toUpperCase() to eliminate a double check - otherwise you'd need
if (text == "Q" || text == "q"){...